angularjs-portal-frame/src/main/webapp/css/buckyless/header.less

```diff
@@ -72,7 +72,7 @@ a#myuw-header img {
 }
 .bucky-nav {
   background:linear-gradient(
-    rgba(20,20,20, 0.8), 
+    rgba(20,20,20, 0.8),
     rgba(20,20,20, 0.8)
```

Comment:
I know that this wasn't you, but does rgba need to be in here twice?  @jhanstra, do you know?  If we don't know, I concede that it's out of scope.